src/main.c

```diff
@@ -2516,13 +2517,16 @@ static int forward_quic_packets(h2o_quic_ctx_t *h3ctx, const uint64_t *node_id,
             if (thread_id == h3ctx->next_cid.thread_id) {
                 assert(h3ctx->acceptor == NULL);
                 /* FIXME forward packets to the newer generation process */
+                H2O_PROBE(H3_PACKET_FORWARD_IGNORED, 0, thread_id);
```

Comment:
This is confusing.

The problem here is that 0 is a valid value as a node_id. 0 is also a valid number as thread_id.

Therefore, when the emitted trace shows `H3_PACKET_FORWARD_IGNORE(0, 0)` we cannot tell if this was an inter- or intra-node forwarding event.

I think we might want to use different probes for indicating these two types of events.

Reply:
> The problem here is that 0 is a valid value as a node_id. 

I thought `node_id == 0` was a special value to indicate that the clustering was not active.
https://github.com/h2o/h2o/blob/master/src/main.c#L3605